latitudesh/cli
# Remove unused auto-generated actions

The CLI generator created commands for all the available API endpoints, but for the beta version we are going to focus only on the most relevant ones.

This PR removes the commands that shouldn't be available on the beta version.

**How to test**

To make sure it didn't break anything, you can check if the build still works, and then validate the endpoints that haven't been removed.
